.gitignore

```diff
@@ -1,4 +1,5 @@
 *.zip
+**/.DS_Store
```

Comment:
The `**/` isn't necessary here.  `.DS_Store` would ignore any file of that name, anywhere in the repo.